src/main/java/com/icthh/xm/tmf/ms/resource/config/ApplicationProperties.java

```diff
@@ -1,13 +1,14 @@
 package com.icthh.xm.tmf.ms.resource.config;
 
 import com.icthh.xm.commons.lep.TenantScriptStorage;
-import java.util.Collections;
-import java.util.List;
 import lombok.Data;
 import lombok.Getter;
 import lombok.Setter;
 import org.springframework.boot.context.properties.ConfigurationProperties;
 
+import java.util.Collections;
```

Comment:
Unnecessary changes, according checkstyle imports orders must be the same as in old version